drivers/sensor/st/lsm6dsv16x/Kconfig

```diff
@@ -38,16 +39,18 @@ config LSM6DSV16X_TRIGGER_NONE
 
 config LSM6DSV16X_TRIGGER_GLOBAL_THREAD
 	bool "Use global thread"
-	depends on GPIO
+	depends on GPIO || I3C
```

Comment:
Maybe this is no longer needed (see #82476)

Reply:
my though here is the the trigger could not use the GPIO, but it could also use the I3C IBI TIR

Reply:
It makes sense.